src/app/ReadClient.h

```diff
@@ -673,8 +673,7 @@ class ReadClient : public Messaging::ExchangeDelegate
 
     System::Clock::Timeout mLivenessTimeoutOverride = System::Clock::kZero;
 
-    bool mIsPeerLIT = false;
-
+    bool mIsPeerLITOperationMode = false;
```

Comment:
```suggestion
    bool mPeerIsOperatingAsLIT = false;
```
if that's what it's supposed to be.

Reply:
agree, updated, thanks